src/settings/SettingsScreen.js

```diff
@@ -82,7 +82,7 @@ export default function SettingsScreen(props: Props): Node {
             key: 'conversation-views-only',
             title: 'Only in conversation views',
             subtitle:
-              'Messages will be automatically marked as read only when viewing a single topic or private message conversation.',
+              'Messages will be automatically marked as read only when viewing a single-topic or direct-message conversation.',
```

Comment:
This revised text suggests these are two kinds of conversations, and there are other kinds of conversations. I don't think that's a way we normally use "conversation".

The original text was intended to mean "viewing a single topic (as opposed to a whole stream), or viewing a (single) PM conversation". Could perhaps be clarified by adding one hyphen: "direct-message conversation" (or previously could have been "private-message conversation"). Or could expand it more wordily if that doesn't seem clear.

In the interest of getting this rename in, probably best to minimize the other wording changes and leave those for a possible followup.

Reply:
Ah OK, right! Thanks for the catch; I'll make it so "private" to "direct" is the only change here.